src/api/cvc4cpp.h

```diff
@@ -3010,6 +3010,15 @@ class CVC4_PUBLIC Solver
    */
   void pop(uint32_t nscopes = 1) const;
 
+
+  /**
+   * Get an interpolant
+   * SMT-LIB: ( get-interpol B )
+   * Requires to enable option 'produce-interpols'.
```

Comment:
Describe the arguments with `@param`.

Reply:
@yoni206 could you take a look that if I added correctly?

Reply:
Yes, looks good. Just two minor things (one below). Can you maybe rename B to something more meaningful? Maybe use `term` or something similar? That's what we typically use in the API.

Reply:
renamed to term.